src/components/PageHeader/CqlLibraryActionCenter/CqlLibraryActionCenter.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState, useEffect } from "react";
+import { SpeedDial, SpeedDialAction } from "@mui/material";
+import DeleteOutlinedIcon from "@mui/icons-material/DeleteOutlined";
+import { MadieDiscardDialog } from "@madie/madie-design-system/dist/react";
+import { CqlLibrary } from "@madie/madie-models";
+import { blue, red } from "@mui/material/colors";
+import { RouteHandlerState, routeHandlerStore } from "@madie/madie-util";
+
+interface PropTypes {
+  canEdit: boolean;
+  library: CqlLibrary;
```

Comment:
I would think the action center is designed only for one library? Do you know if it will be used in library list?

Reply:
This Action center is not present at the cql library list level. it is for only showing the action center inside the library so there wouldn't be multiple selection of libraries